src/pages/understand-stacks/managing-accounts.md

```diff
@@ -75,15 +75,15 @@ const stacksAddress = getAddressFromPrivateKey(
   TransactionVersion.Testnet // remove for Mainnet addresses
 );
 
-...
-
+async function getAccountInfo() {
   const accounts = new AccountsApi(apiConfig);
```

Comment:
I think this was is used in other methods, I'd suggest initializing it outside of this method. so the other examples could work with copy&paste

Reply:
Fixed